index.bs

```diff
@@ -392,7 +392,7 @@ below.
   ::
    <p>Do nothing.
    <p class="note">The connection is already closing or is already closed. If it has not already, a
-   <a event><code>close</code></a> event will eventually fire <a href="#closeWebSocket">as described
+   <a event>`close`</a> event will eventually fire <a href="#closeWebSocket">as described
```

Comment:
This should be able to be something like `{{WebSocket/close}}` or `{{WebSocket/close!!event}}`, although you'll need to add the appropriate `for="WebSocket"` at the definition site.

Reply:
All these events are defined in HTML and shared with other features. Or do you mean that we should un-share them?